src/cli/tc.rs

```diff
@@ -104,11 +108,37 @@ pub async fn init(file_path: &str) -> Result<()> {
 pub fn display_schema(blueprint: &Blueprint) {
   Fmt::display(Fmt::heading(&"GraphQL Schema:\n".to_string()));
   let sdl = blueprint.to_schema();
-  Fmt::display(print_schema::print_schema(sdl));
+  Fmt::display(format!("{}\n", print_schema::print_schema(sdl)));
```

Comment:
these changes are out of the scope of #416

```suggestion
  Fmt::display(print_schema::print_schema(sdl));
```

Reply:
I got that this meant to break the line, but we need to do so not here but where we'll print the other message

Reply:
this review is still valid @tobihans

Reply:
this review is still valid @tobihans